ROADMAP.rst

```diff
@@ -0,0 +1,22 @@
+=================================
+ Roadmap for Next-gen Jobbergate
+=================================
+
+This explains the roadmap for migrating from the legacy jobbergate apps to the
+next generation of Jobbergate.
+
+
+Work Items::
+
+    [x] Add capability to dump full json output from legacy jobbergate-cli
+    [x] Rename jobbergate-api-fastapi to jobbergate
+    [x] Move jobbergate-api-fastapi code to sub-project in jobbergate/jobbergate-api
+    [ ] Migrate jobbergate-cli v2 branch to sub-project in jobbergate/jobbergate-cli
```

Comment:
What is the meaning of `sub-project`? A directory? A git submodule?

Reply:
A subdirectory which is a python project unto itself. I considered submodules, but that can just cause a lot more complication than I want to jump into now.

Reply:
Would you mind rewording then to be explicit it is a directory? This might be a cause of confusion in the future.